demos/electron/index.js

```diff
@@ -25,7 +25,7 @@ _gaq.push(['_trackPageview']);
 
 (function() {
 	var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-	ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+	ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'https://www') + '.google-analytics.com/ga.js';
 	var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
 })();
```

Comment:
Not too familiar with electron, but what's the deal with doing `(function() { ... })();`? Do these lines need to be wrapped in a self-invoking function?

Reply:
Honestly that entire block should be removed.  This demo started from the example at https://oss.sheetjs.com/sheetjs/ which had the GA code

Reply:
Yep these need to attached to the DOM. I guess they don't necessarily have to be wrapped in self-invoking function and could be put in the global scope of the script but the functions add some grouping to what each thing is doing. And I didn't want to change the existing structure.